packages/@coorpacademy-components/src/molecule/playlist-detail-cover/index.js

```diff
@@ -0,0 +1,56 @@
+import React, {useMemo, useCallback} from 'react';
```

Comment:
do we also need a default render if there is no images props passed to the component?

Reply:
I don't think so, images props is required, we do have fallback images if no url is provided though